guides/v2.3/reference/cli/magento.md

```diff
@@ -1,11 +1,24 @@
 ---
 layout: full-width
+title: 'Command-line tools: bin/magento'
```

Comment:
Is there a reason why this isn't symlinked to 2.2? It looks identical to the 2.2 file.

Reply:
This uswe different data files, so the following line is different:
```
{% assign file = site.data.codebase.v2_2.open-source.bin-magento %}
```
```
{% assign file = site.data.codebase.v2_3.open-source.bin-magento %}
```